src/inmanta/module.py

```diff
@@ -263,7 +264,7 @@ def __init__(self, msg: str, validation_error: Optional[ValidationError] = None)
     def _extend_msg_with_validation_information(cls, msg: str, validation_error: ValidationError) -> str:
         errors = validation_error.errors()
         if errors:
-            msg += "\n" + textwrap.indent(display_errors(errors), " " * 2)
+            msg += "\n" + textwrap.indent(str(validation_error), " " * 2)
```

Comment:
This change has an effect on how the error message looks. I think it's acceptable but if you disagree I can fix it. It would require more manual formatting than before.

Previous output:
```
Metadata defined in ./project.yml is invalid:
  pip -> use-system-config
    Input should be a valid boolean, unable to interpret input (type=bool_parsing)
```

New output:
```
Metadata defined in ./project.yml is invalid:
  1 validation error for ProjectMetadata
  pip.use-system-config
    Input should be a valid boolean, unable to interpret input [type=bool_parsing, input_value='hello', input_type=str]
      For further information visit https://errors.pydantic.dev/2.5/v/bool_parsing
```

CC @arnaudsjs @Hugo-Inmanta

Reply:
This is fine by me :+1: